Gems/Maestro/Code/Source/Cinematics/SceneNode.cpp

```diff
@@ -1092,16 +1093,20 @@ void CAnimSceneNode::InitializeTrackDefaultValue(IAnimTrack* pTrack, const CAnim
 /*static*/ IAnimSequence* CAnimSceneNode::GetSequenceFromSequenceKey(const ISequenceKey& sequenceKey)
 {
     IAnimSequence* retSequence = nullptr;
-    if (gEnv && gEnv->pMovieSystem)
+
+    IMovieSystem* m_movieSystem = nullptr;
```

Comment:
(style) `m_movieSystem` is not a class member
```suggestion
    IMovieSystem* movieSystem = nullptr;
```